### PR TITLE
Add jvm heap as a cfn parameter.

### PR DESCRIPTION
**Pull request by kowshikn**
*Issue #304*

*Description of changes:*

* Added the heap as a parameter and set it as JAVA_OPTS.
* Use same workers, buffer and batch size.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml [hunk 1]
```diff
@@ -144,6 +158,8 @@ Resources:
         # Script to download and run Data Prepper
         Fn::Base64: !Sub |
           #!/bin/bash
+          export JAVA_OPTS="-xms${HeapInMb}m -xmx${HeapInMb}m"
```
Comment (wrijeff) [inline deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml:161]: The first x's need to be capitalized: `-Xms256m -Xmx256m` (ref https://docs.oracle.com/cd/E15523_01/web.1111/e13814/jvm_tuning.htm#PERFM161)

Comment (kowshikn) [reply deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml:161]: Lol That was dumb :(

Comment (wrijeff) [reply deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml:161]: No problem, you found a bug! Created issue #313

**Comment by kowshikn** [comment]
```
/usr/bin/java
Found openjdk version  of 11.0
Unrecognized option: -xms256m
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

Appended the error to the same data-prepper.out

File: deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml (hunk 1 above)
Comment (kowshikn) [reply deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml]: I actually added the fix to append error to the same output file.

File: deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml [hunk 2]
```diff
@@ -158,14 +158,14 @@ Resources:
...
-          export JAVA_OPTS="-xms${HeapInMb}m -xmx${HeapInMb}m"
+          export JAVA_OPTS="-Xms${HeapInMb}m -Xmx${HeapInMb}m"
           echo $JAVA_OPTS > /var/log/java_opts.txt
           export RELEASE=opendistroforelasticsearch-data-prepper-${DataPrepperVersion}-linux-x64
           yum install java-11-amazon-corretto-headless -y
           wget https://github.com/opendistro-for-elasticsearch/data-prepper/releases/download/v${DataPrepperVersion}/$RELEASE.tar.gz -O /tmp/$RELEASE.tar.gz
           tar -xzf /tmp/$RELEASE.tar.gz --directory /usr/local/bin
           /opt/aws/bin/cfn-init -v --stack ${AWS::StackId} --resource EC2Instance --region ${AWS::Region} --configsets default
-          nohup /usr/local/bin/$RELEASE/data-prepper-tar-install.sh /etc/data-prepper/config.yaml > /var/log/data-prepper.out &
+          nohup /usr/local/bin/$RELEASE/data-prepper-tar-install.sh /etc/data-prepper/config.yaml > /var/log/data-prepper.out 2>&1
```
Comment (wrijeff) [inline deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml:168]: Thanks for updating 👍

**Review by wrijeff — APPROVED** [review]

**Pull request merged**